monitoring/pra/alerts.yaml

```diff
@@ -0,0 +1,79 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: kafka_connect_src_job
+    type: config
+    value: artesca-data-dr-source-base-queue-connector-metrics
+  - name: kafka_connect_sink_job
+    type: config
+    value: artesca-data-dr-base-queue-connector-metrics
+  - name: dr_sink_instance
+    type: config
+    value: artesca-data-dr
+  - name: RTO
+    type: constant
+    value:  86400 # 24 hours
+  - name: mongo_jobs
+    type: config
+    value: zenko/data-db-mongodb-sharded-shard.*
+
+groups:
+- name: PraAlerts
+  rules:
+
+  - alert: DrResourcePausedWhileOtherRunning
+    expr: |
+      (sum(up{job="${kafka_connect_src_job}", namespace="${namespace}"}) OR vector(0)) != (sum(up{job="${kafka_connect_sink_job}", namespace="${namespace}"}) OR vector(0))
+    for: 1m
+    labels:
+      severity: warning
+    annotations:
+      summary: 'DR Resource Paused While Other Running'
+      description: 'One site s DR resource is paused while the other is not. This could lead to data inconsistency between sites.'
+
+  - alert: DrResourcePausedForTooLong
+    expr: |
+      (sum(up{job="${kafka_connect_src_job}", namespace="${namespace}"}) OR vector(0)) == 0
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: 'DR resource paused for too long'
+      description: 'The DR resource has been paused for more than 5 minutes. This could lead to data loss if not addressed promptly.'
```

Comment:
> This could lead to data loss if not addressed promptly.

this is not a **data loss** !
and pausing for 5 minutes is not an issue: whenever pause is used for "real", it will probably be longer than 5 minutes. There is no urgency at this time
→ having a warning as "indication" is fine, but this is just an indication. No data loss or "prompt" action.